src/core/lib/gprpp/time.h

```diff
@@ -116,6 +148,21 @@ class Timestamp {
   explicit constexpr Timestamp(int64_t millis) : millis_(millis) {}
 
   int64_t millis_ = 0;
+  static GPR_THREAD_LOCAL(Timestamp::Source*) source_;
```

Comment:
Should we have some naming convention for thread-locals, like the way we use a `g_` prefix for globals?  The reason I ask is that `source_` implies that this is a normal class data member, which is not the case here, and it took me a while to figure out where it was coming from.

Reply:
Maybe... but I'll note our `g_` is at odds with the normal Google style guide.

Reply:
Yeah, I realize that.  I suspect the main reason that the Google style guide doesn't have a naming convention for this is that they discourage use of globals to begin with.  We tend to use a lot of them in gRPC, and it seems like we're now starting to use more thread-locals too.

If nothing else, suggest calling this something like `thread_local_time_source_`.

Reply:
done